src/main.py

```diff
@@ -166,33 +181,70 @@ def decision(update, context):
     context.bot.send_animation(chat_id=update.message.chat_id, animation=data["image"], caption=data["answer"])
 
 
-def subredditImg(subreddit, offset=0, count=5):
-
-    imageFileEndings = [".png", ".jpg", ".jpeg", ".webp", ".gif"]
-
-    reddit = praw.Reddit(client_id=REDDIT_BOT_ID, client_secret=REDDIT_BOT_SECRET, user_agent=REDDIT_USER_AGENT)
-
+def get_post_type(post):
+    post_type = RedditPostTypes.undefined
+    if post.selftext != "":
+        post_type = RedditPostTypes.text
+    else:
+        for ending in REDDIT_IMAGE_FILE_ENDINGS:
+            if post.url.endswith(ending):
+                post_type = RedditPostTypes.image
+                break
+        for ending in REDDIT_ANIMATION_FILE_ENDINGS:
+            if post.url.endswith(ending):
+                post_type = RedditPostTypes.animation
+                break
+        for video_site in REDDIT_VIDEO_SITES:
+            if video_site in post.url:
+                post_type = RedditPostTypes.video
+                break
+    return post_type
+
+
+def get_subreddit_images(subreddit, offset=0, count=5):
     images = []
-
+    reddit = praw.Reddit(client_id=REDDIT_BOT_ID, client_secret=REDDIT_BOT_SECRET, user_agent=REDDIT_USER_AGENT)
     for post in reddit.subreddit(subreddit).hot(limit=count):
-        for ending in imageFileEndings:
-            if str(post.url).endswith(ending):
-                images.append(post.url)
+        if get_post_type(post) == RedditPostTypes.image:
+            images.append(post.url)
     return images
 
 
-def sendSubredditImages(subreddit_display_name, update, context):
+def send_subreddit_posts(subreddit, update, context, offset=0, count=5):
```

Comment:
For the future, we should use the same code for the posting list iteration for both `send_subreddit_posts` and `get_subreddit_images`